package.json

```diff
@@ -2,13 +2,14 @@
   "name": "unistyle",
   "version": "0.1.3",
   "description": "Write modular and scalable CSS using the next version of ECMAScript.",
-  "main": "src/index.js",
+  "main": "build.js",
   "bin": {
     "unistyle": "bin/unistyle"
   },
   "scripts": {
-    "lint": "xo",
+    "lint": "xo --ignore build.js || true",
```

Comment:
Should go in the XO config: https://github.com/joakimbeng/unistyle/pull/3/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L37